gutterball/pom.xml

```diff
@@ -821,6 +822,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.candlepin</groupId>
+      <artifactId>candlepin-common</artifactId>
+      <version>2.0.0</version>
+      <classifier>complete</classifier>
```

Comment:
Does every common update require a manual version update in gutterball/candlepin?